code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -237,3 +237,8 @@
 	. = ..()
 	if(.)
 		update_icons()
+
+/mob/living/simple_animal/drone/cogscarab/Life()
```

Comment:
this is not an appropriate fix.

Reply:
what would be then

Reply:
Maybe `handle_status_effects()`?

Reply:
that's also inside of Life, though.
it should be in update_canmove.